skiko/src/jvmMain/kotlin/org/jetbrains/skiko/SkikoProperties.kt

```diff
@@ -89,6 +89,17 @@ object SkikoProperties {
 
     val macOsOpenGLEnabled: Boolean get() = getProperty("skiko.macos.opengl.enabled")?.toBoolean() ?: false
 
+    private fun getProperty(key: String): String? {
+        return classLoaderProperties?.getProperty(key) ?: System.getProperty(key)
+    }
+
+    private val classLoaderProperties by lazy {
+        val res = SkikoProperties::class.java.classLoader.getResourceAsStream("skiko.properties") ?: return@lazy null
```

Comment:
Is this `skiko.properties` something new or it was used somewhere before?

Reply:
It is new.